src/ripple/app/tests/PayChan_test.cpp

```diff
@@ -0,0 +1,689 @@
+        boost::optional<NetClock::time_point> cancelAfter = boost::none,
+        boost::optional<std::uint32_t> dstTag = boost::none)
+    {
+        using namespace jtx;
+        Json::Value jv;
+        jv[jss::TransactionType] = "PaymentChannelCreate";
+        jv[jss::Flags] = tfUniversal;
+        jv[jss::Account] = account.human ();
+        jv[jss::Destination] = to.human ();
+        jv[jss::Amount] = amount.getJson (0);
+        jv["SettleDelay"] = settleDelay.count ();
+        jv["PublicKey"] = strHex (pk.slice ());
+        if (cancelAfter)
+            jv["CancelAfter"] = cancelAfter->time_since_epoch ().count ();
+        if (dstTag)
+            jv["DestinationTag"] = *dstTag;
+        return jv;
+    }
+
+    static
+    Json::Value
+    fund (jtx::Account const& account,
+        uint256 const& channel,
+        STAmount const& amount,
+        boost::optional<NetClock::time_point> expiration = boost::none)
+    {
+        using namespace jtx;
+        Json::Value jv;
+        jv[jss::TransactionType] = "PaymentChannelFund";
+        jv[jss::Flags] = tfUniversal;
+        jv[jss::Account] = account.human ();
+        jv["Channel"] = to_string (channel);
+        jv[jss::Amount] = amount.getJson (0);
+        if (expiration)
+            jv["Expiration"] = expiration->time_since_epoch ().count ();
+        return jv;
+    }
+
+    static
+    Json::Value
+    claim (jtx::Account const& account,
+        uint256 const& channel,
+        boost::optional<STAmount> const& balance = boost::none,
+        boost::optional<STAmount> const& amount = boost::none,
+        boost::optional<Slice> const& signature = boost::none,
+        boost::optional<PublicKey> const& pk = boost::none)
+    {
+        using namespace jtx;
+        Json::Value jv;
+        jv[jss::TransactionType] = "PaymentChannelClaim";
+        jv[jss::Flags] = tfUniversal;
+        jv[jss::Account] = account.human ();
+        jv["Channel"] = to_string (channel);
+        if (amount)
+            jv[jss::Amount] = amount->getJson (0);
+        if (balance)
+            jv["Balance"] = balance->getJson (0);
+        if (signature)
+            jv["Signature"] = strHex (*signature);
+        if (pk)
+            jv["PublicKey"] = strHex (pk->slice ());
+        return jv;
+    }
+
+    void
+    testSimple ()
+    {
+        testcase ("simple");
+        using namespace jtx;
+        using namespace std::literals::chrono_literals;
+        Env env (*this, features (featurePayChan));
+        auto const alice = Account ("alice");
+        auto const bob = Account ("bob");
+        env.fund (XRP (10000), alice, bob);
+        auto const pk = alice.pk ();
+        auto const settleDelay = 100s;
+        env (create (alice, bob, XRP (1000), settleDelay, pk));
+        auto const chan = channel (*env.current (), alice, bob);
+        expect (channelBalance (*env.current (), chan) == XRP (0));
+        expect (channelAmount (*env.current (), chan) == XRP (1000));
+
```

Comment:
Should probably check alice's balance after create, to verify the amount was sequestered.